src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -113,10 +119,16 @@
       "Size": 3943
     },
     "assemblies/System.Security.Cryptography.Algorithms.dll": {
-      "Size": 6815
+      "Size": 37063
+    },
+    "assemblies/System.Security.Cryptography.Encoding.dll": {
+      "Size": 13705
     },
     "assemblies/System.Security.Cryptography.Primitives.dll": {
-      "Size": 2973
+      "Size": 7317
+    },
+    "assemblies/System.Security.Cryptography.X509Certificates.dll": {
+      "Size": 52650
```

Comment:
So I'm guessing an app using `HttpClient` will bring in these assemblies by default now? Is the linker preserving every property on `AndroidMessageHandler`?

Maybe we need to look into why that's happening.

Reply:
I thought that the linker would remove these assemblies when the callback isn't set but apparently that's not happening. I need to check the .dll that are produced in these tests and what's trimmed away and what's left in there.

Reply:
I was able to figure out how to make the linker trim away these dependencies when the callback is not set, I pushed to the PR and @simonrozsival will update the .apkdesc files with the new values.